src/org/traccar/protocol/EelinkProtocolDecoder.java

```diff
@@ -97,28 +114,72 @@ private String decodeAlarm(Short value) {
     }
 
     private void decodeStatus(Position position, int status) {
+
+        // GPS - 0 bit
```

Comment:
Why write unnecessary comments into the position table that no one needs to refer to unless they look at the decoder when they are trying to work out what is happening.
I am using the words that are in their spec file to clarify. You are nitpicking on the this.